src/pybind/mgr/dashboard/frontend/src/app/shared/services/summary.service.ts

```diff
@@ -18,26 +18,32 @@ export class SummaryService {
   // Observable streams
   summaryData$ = this.summaryDataSource.asObservable();
 
+  pooling: number;
```

Comment:
Did you mean [polling](https://en.wikipedia.org/wiki/Polling_(computer_science))? As [pooling](https://en.wikipedia.org/wiki/Pooling_(resource_management)) refers more to a term used in finance as Wikipedia enlightened me.

Reply:
Good spotting! I concur, please fix the spelling and replace "pooling" with "polling", @tspmelo

Reply:
Done.